games-fps/alephone/alephone_marathon-20150907.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="Marathon 1 game assets"
+DESCRIPTION="Alien forces have boarded the interstellar colony ship \
+Marathon. The situation is dire. As a security officer onboard, it is your \
+duty to defend the ship and its crew.
+
+Experience the start of Bungie\'s iconic trilogy with Marathon. This release \
+uses the original Marathon data files for the most authentic experience \
+outside of a classic Mac or emulator."
+HOMEPAGE="https://alephone.lhowon.org/"
+COPYRIGHT="1991-2001 Bungie Software
+	Aleph One Developers"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20150620/Marathon-20150620-Data.zip"
+SOURCE_DIR="Marathon"
+CHECKSUM_SHA256="774cb40c9bde33a5d23146e82f593ac001edd0ee4d62e5b0445953a95fd44f31"
+ADDITIONAL_FILES="alephone.rdef.in"
+
+ARCHITECTURES="any"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+    commandSuffix=
+fi
+
+PROVIDES="
+	alephone_marathon
+	cmd:marathon$commandSuffix
```

Comment:
```diff
-# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
-commandSuffix=$secondaryArchSuffix
-if [ "$targetArchitecture" = x86_gcc2 ]; then
-    commandSuffix=
-fi
-
 PROVIDES="
 	alephone_marathon
-	cmd:marathon$commandSuffix
+	cmd:marathon
 	"
```

The reason why we have a `$commandSuffix` in `alephone-20180414.recipe` is to make sure someone running `hp -S alephone_x86` on x86_64 hybrid will not produce a package that would conflict with the `alephone` for x86_64 primary arch.
Here, for the `"any"` arch, we don't need (nor want) a `$commandSuffix` in `alephone_marathon-20150907.recipe`.

(Same comments for `alephone_marathon2-20150907.recipe` and `alephone_marathoninfinity-20150907.recipe`.)